public/components/main/report_definition_details/report_definition_details.tsx

```diff
@@ -110,11 +108,11 @@ export function ReportDefinitionDetails(
               alignItems="flexEnd"
               gutterSize="xs"
             >
-              <EuiFlexItem></EuiFlexItem>
-              <EuiFlexItem></EuiFlexItem>
-              <EuiFlexItem></EuiFlexItem>
-              <EuiFlexItem></EuiFlexItem>
-              <EuiFlexItem></EuiFlexItem>
+              <EuiFlexItem />
+              <EuiFlexItem />
+              <EuiFlexItem />
+              <EuiFlexItem />
+              <EuiFlexItem />
```

Comment:
Could you change line 142, 143 as well just to be consistent?

Reply:
Good catch, missed that